docs/sources/k6/next/using-k6/secret-source/_index.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Secret Source
+description: 'Secret source are used so k6 can get and use secrets in a secure way'
+weight: 14
+---
+
+# Secret source
+
+Secret sources are a way for k6 to acquire secrets to be used with k6.
+
+The secrets are made available through the `[k6/secrets`]() JS API and will be redacted from any logs.
+
+## Configure secret sources
+
+Currently the only way to configured secret sources is through the `--secret-source` cli flag. Multiple secret sources are configurable at the same time
+
+## Secret sources
+
+Current build-in secret sources are limited and are mostly meant for local testing:
```

Comment:
It might confuse people, why are we making a public API if they shouldn't use it? I think `file` is there to be used, or not?

```suggestion
Current build-in secret sources are limited and some are meant for local testing:
```